src/styles/font-jrkf/iconfont.js

```
@@ -0,0 +1,68 @@
+              document.addEventListener('DOMContentLoaded', t, !1))
+          : document.attachEvent &&
+            ((i = h),
+            (v = o.document),
+            (m = !1),
+            c(),
+            (v.onreadystatechange = function () {
+              'complete' == v.readyState && ((v.onreadystatechange = null), s())
+            }))
+    }
+    function s() {
+      m || ((m = !0), i())
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)